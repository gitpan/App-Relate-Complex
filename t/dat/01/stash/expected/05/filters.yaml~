---
:skipdull:
  description: Screen out some uninteresting files.
  method: skip_any
  modifiers: x
  terms:
    - '~$'
    - /\#
    - /\.\#
    - ',v$'
    - /CVS$
    - /CVS/
    - /RCS$
    - \.elc$
