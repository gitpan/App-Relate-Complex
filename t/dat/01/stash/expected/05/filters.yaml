---
:skipdull:
  description: Screen out some uninteresting files.
  method: skip_any
  modifiers: x
  terms:
    - '~$'
    - /\#
    - /\.\#
    - ',v$'
    - /CVS$
    - /CVS/
    - /RCS$
    - \.elc$
bleach:
  description: gets rid of unsightly blemishes
  method: skip_any
  modifiers: x
  terms:
    - '^.*$'
